From 5f1e8f211050870860005b171578e6ee8cfb1f84 Mon Sep 17 00:00:00 2001
From: Tad <tad@spotco.us>
Date: Fri, 27 Nov 2020 13:47:56 -0500
Subject: [PATCH] dnsmasq: only listen on br-lan interface

dnsmasq should not listen on all interfaces, restrict it to br-lan.

Note: this will break default config if your LAN isn't br-lan.
Signed-off-by: Tad <tad@spotco.us>
---
 package/network/services/dnsmasq/files/dnsmasq.conf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/package/network/services/dnsmasq/files/dnsmasq.conf b/package/network/services/dnsmasq/files/dnsmasq.conf
index bf5816be56..24440b5e1c 100644
--- a/package/network/services/dnsmasq/files/dnsmasq.conf
+++ b/package/network/services/dnsmasq/files/dnsmasq.conf
@@ -35,3 +35,5 @@
 # "bert" another name, bertrand
 # The fields are <cname>,<target>
 #cname=bertand,bert
+
+interface=br-lan
-- 
2.31.1

